job_server/src/routes/v1.ts

```diff
@@ -21,6 +22,11 @@ export default function (app: Router) {
     app.delete("/mailing_list", deleteFromMailingList);
     app.options("/mailing_list", corsAllowMethods("POST,DELETE"));
 
+    app.get("/job", getJob);
```

Comment:
@rupertotorres1 is updating the routes so that everything in the job server starts with the same path, so the paths will probably have to be updated after that

Reply:
@rupertotorres1 is this in a branch somewhere? Or can you just tell me what the job server prefix would be?

Reply:
Hey sorry for the delay. I was thinking the prefix would be "job" or "jobs".